### PR TITLE
added buttons, resolved some simple style issues

### PR DESCRIPTION
**Pull request by gurjit03**

### CODE REVIEW

**Comment by alexpricedev** [comment]
Any reason you removed the `lock` file?

**Comment by gurjit03** [comment]
I haven't removed it actually, I will have a look

**Comment by alexpricedev** [comment]
Maybe you have it in a global `.gitignore` file somewhere up the directory tree?

**Comment by gurjit03** [comment]
Having a .gitignore file should not over shadow/ over ride the current .gitignore file in the project ?

**Comment by gurjit03** [comment]
Having a .gitignore file should not over shadow/ over ride the current .gitignore file in the project ? Though i dont have it, i have checked it.

**Comment by alexpricedev** [comment]
yeah i guess it's still there! Just looked like it had gone from 
screen shot 2017-11-25 at 15 17 21

File: src/Tooltip.js [hunk 1]
```diff
@@ -22,19 +31,29 @@ const afterStyles = {
...
     style={toolTipStyles}
   >
-    { emailAddress }
-    <span
+  <div className={className + '__tooltip-head'}>{ emailAddress }</div>
+  <div className={className + '__tooltip-body'}>
+    <button
+      className={className + '__tooltip--copy-email-address __tooltip-body-btn'}
+      style={toolTipBodyBtnStyles}
+      >Copy Email Address To ClipBoard</button>
```
Comment (alexpricedev) [inline src/Tooltip.js:46]: I prefer this code style:

```
<button
  className={className + '__tooltip--copy-email-address __tooltip-body-btn'}
  style={toolTipBodyBtnStyles}
>
  Copy Email Address To ClipBoard
</button>
```

I'll setup linting sometime in a new PR

**Review by alexpricedev — APPROVED** [review]
Can you change the JSX to match the style I suggested? Looks great otherwise 🚀

File: src/Tooltip.js (hunk 1 above)
Comment (gurjit03) [reply src/Tooltip.js:46]: yes sure I will do for sure :) :+1:

**Pull request merged**